deploy/operator/setup_lso.sh

```diff
@@ -23,22 +23,22 @@ function install_lso() {
   catalog_source_name="redhat-operators"
 
   OC_VERSION_MAJOR_MINOR=$(oc version -o json | jq --raw-output '.openshiftVersion' | cut -d'.' -f1-2)
-  if [[ ${OC_VERSION_MAJOR_MINOR} == "4.13" && "${DISCONNECTED}" != true ]]; then
-      # LSO has not been published to the 4.13 redhat-operators catalog, so
-      # it cannot be installed on OpenShift 4.13. Until this is resolved,
-      # we explicitly install the 4.12 catalog as redhat-operators-v4-12
-      # and then subscribe to the LSO version from the 4.12 rather than the 4.13 catalog.
+  if [[ ${OC_VERSION_MAJOR_MINOR} >= "4.14" && "${DISCONNECTED}" != true ]]; then
```

Comment:
```suggestion
  if [[ ${OC_VERSION_MAJOR_MINOR} == "4.14" && "${DISCONNECTED}" != true ]]; then
```
```
[2023-06-28 11:36:28] /home/assisted-service/deploy/operator/setup_lso.sh: line 26: syntax error in conditional expression
```

and I don't think we want to hide the issue once 4.15 is out, I think it would be better to follow the issue, and detect if the catalog is released or not.